hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/LogEventViewGenerator.java

```diff
@@ -76,10 +77,12 @@ private String getSummary(Attributes attributes) {
       return null;
     }
     Map<String, AttributeValue> attributeValueMap = attributes.getAttributeMap();
-    Optional<String> summary =
+    Optional<String> summaryKey =
         SUMMARY_KEYS.stream().filter(attributeValueMap::containsKey).findFirst();
-    return summary.orElseGet(
-        () -> attributeValueMap.entrySet().stream().findFirst().get().getValue().getValue());
+
+    return summaryKey.isPresent()
```

Comment:
Doing `attributeValueMap.entrySet().stream().findFirst().get().getValue().getValue()` is bad. What if there is no attribute present?

nit: 

```
return summaryKey
    .map(summary -> attributeValueMap.get(summary).getValue())
    .orElse(
        attributeValueMap.entrySet().stream()
            .findFirst()
            .map(attribute -> attribute.getValue().getValue())
            .orElse(null));
```

Reply:
> Doing `attributeValueMap.entrySet().stream().findFirst().get().getValue().getValue()` is bad. What if there is no attribute present?

That's not a possibility because of this check in the first line of the method

```
if (isEmpty(attributes)) {
      return null;
    }
```

Let me use `orElse` for the optional